src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let totalSum = 0;
+
+  const adder = (x) => {
```

Comment:
```suggestion
  const adder = (num) => {
```
Better to use more meaningful names